src/HaskellWorks/Data/Uri/IO/Lazy.hs

```diff
@@ -9,6 +9,7 @@ module HaskellWorks.Data.Uri.IO.Lazy
   , firstExistingResource
   , headS3Uri
   , writeResource
+  , writeResource'
```

Comment:
Where this function is used?

Reply:
In any project that wants to write via filename